test/_init_tests.js

```diff
@@ -188,6 +188,11 @@ afterEach(() => {
     sandboxes.splice(0, sandboxes.length).forEach(sb => sb.restore());
     throw new Error('You forgot to restore your sandbox!');
   }
+  if (global.sandbox) {
```

Comment:
It gets here just fine. I tested it. The `restore` part here is not really needed per code above, so if that's in the way - I can remove it. But I need an error and `delete`.